xray/tutorial.py

```diff
@@ -0,0 +1,61 @@
+'''
+Useful for:
+
+* users learning xray
+* building tutorials in the documentation.
+
+'''
+
+import os
+
+import requests
+
+from .backends.api import open_dataset
+
+
+# Borrowed from Seaborn
+def load_dataset(name='ncep_temperature_north-america_2013-14.nc',
```

Comment:
added, but skipping for now until we can make tests conditional on network access.